tests/test_cli.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from typer.testing import CliRunner
+
+from seagap.cli import app
+
+runner = CliRunner()
+
+
+@pytest.mark.parametrize(
+    "commands,expected_stdouts",
+    [
+        (
+            [],
+            [
+                "GNSS-A Processing in Python",
```

Comment:
It would be easier to maintain if you just assert that the number of expected items so that this doesn't have to be update with every change to the help. I.e. all you want is to know that each command is showing some help. The actual content of the help string is not likely to break.